delete_single_ocp.yml

```diff
@@ -0,0 +1,37 @@
+---
+# This playbook is used to destroy single OCP cluster and fips.
+# this playbook is usefull when you want to re create the OCP without
+# redeploying the OSP
+#
+# Example usage
+#
+# ansible-playbook -i host delete_single_ocp.yml
```

Comment:
better also add
ansible-playbook -i ~/.infrared/.workspaces/active/hosts host delete_single_ocp.yml
As a later enhancement make to it generic to delete all or requested OCP clusters.